test/parallel/test-url-fileurltopath.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+const { isWindows } = require('../common');
+const assert = require('assert');
+const url = require('url');
+
+// Input must be string or URL
+assert.throws(() => url.fileURLToPath(null));
```

Comment:
Linter is complaining that there is no second argument in `assert.throws()` here and elsewhere.  (You can run the JS linter locally with `make lint-js` or, I believe, `vcbuild lint-js`. Running `make test` or, I think, `vcbuild test` should run the linter as well.) So this instance might be:

```js
assert.throws(() => { url.fileURLToPath(null); }, { code: 'ERR_INVALID_ARG_TYPE' });
```

We require the second argument because changing the `code` of the error is considered a breaking change. So we want the test to validate the `code` property at least.

Reply:
I added the second args. Thanks!